bin/MadGraph5_aMCatNLO/Template/_customizecards.dat

```diff
@@ -0,0 +1,13 @@
+set param_card  frblock  1  4.000000e+02
```

Comment:
there are a lot of filenames which are not good...

Reply:
here we define the mass of dark matter. So it is necessary

Reply:
yes but are you sure you really want these info to be stored in a file called
bin/MadGraph5_aMCatNLO/Template/_customizecards.dat ?
probably not...

Reply:
If these are not stored in the _customizecards.dat, then where should they be stored?
I am not expert on MG5, so if you have a better idea, please let me know, otherwise, I see no problem of defining them in another file